src/particles/ParticleEmitter.ts

```diff
@@ -9,7 +9,8 @@ interface Particle {
   color: string
   radius: number
   orientation: number
-  spawnTime: number
+  speed: number
```

Comment:
This is the one major logic change in this PR: I noticed that the same particle was moving with random displacement per frame, which is a little weird. So I gave each particle a fixed velocity, and tuned the parameters of existing emitters to best simulate previous behavior.